src/adapters/scene.js

```diff
@@ -145,10 +147,29 @@ Scene.prototype.initMapBox = function() {
       }, this.DOUBLE_TAP_DELAY_MS);
     });
 
+    // Long touch on mobile (500ms)
+    // Custom implementation because the contextmenu event isn't supported by MapBox.
+    // Long touch is initiated with a touchstart event
+    // It is canceled if many fingers touch the map, or if one of these events is initiated:
+    // touchmove, touchcancel, touchend, pointerdrag, pointermove, moveend, gesturestart, gesturechange, gestureend
     if (isMobileDevice()) {
-      this.mb.on('contextmenu', e => {
-        this.clickOnMap(e.lngLat, null, { longTouch: true });
+      this.mb.on('touchstart', e => {
+        if (e.originalEvent.touches.length === 1) {
+          longTouchTimeout = setTimeout(() => {
+            this.clickOnMap(e.lngLat, null, { longTouch: true });
+          }, 500);
```

Comment:
can we use/modify DOUBLE_TAP_DELAY_MS ?

Reply:
That would be interesting indeed to explore the possible relationship with these two values. Can they be the same? How did you decide about 500ms?
I would just avoid modifying the current value of DOUBLE_TAP_DELAY_MS as this is the value used by MapBox internal implementation and it seems right UX-wise.

Reply:
Long press duration is not a magic value unfortunately. It's specific to Mobile OS's and on certain devices you can even modify the long touch duration in the OS settings, but afaik there's no way to read it in JS.
I chose 500ms as it seems to be the default value on Android.
I would not recommend using DOUBLE_TAP_DELAY_MS as it has a wrong name and 300ms seems too short in my opinion.
However I recommend creating a LONG_TOUCH_DELAY_MS constant equal to 500 or any other value you prefer.
@bbecquet @G-Ray

Reply:
So what I understand is this number comes from knowledge of the mobile platform, and you use it so our app behavior is consistent with user interaction on native apps.
I agree defining a LONG_TOUCH_DELAY_MS const would be nice. And if you have a link to a doc or example justifying its 500ms value, you can add it in comments like we did for DOUBLE_TAP_DELAY_MS.